# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gegl/hotfix.patch.cross
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/meson.build
+++ b/meson.build
@@ -703,7 +703,7 @@ subdir('po')
 subdir('docs')
 
 
-if host_machine.system() != 'emscripten'
+if host_machine.system() != 'emscripten' and not meson.is_cross_build()
   nm = find_program('nm', required: false)
 
   # Verify .def files for Windows linking (and for macOS too when -Wl,-exported_symbols_list is used).
